app-codepipeline/vars.tf

```diff
@@ -8,9 +8,30 @@ variable "name_prefix" {
   }
 }
 
+variable "ecr_namespace" {
```

Comment:
Can you add these new vars to the tfvars.template file?

Reply:
_done_